packages/tests/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "dependencies": {
-    "@ethersproject/cli": "^5.5.0",
-    "@ethersproject/testcases": "^5.5.0",
+    "@ethersproject/cli": "5.5.0",
```

Comment:
We shouldn't need the `cli` as dependency

Reply:
`cli` package is removed